jscomp/test/obj_magic_test.js

```diff
@@ -22,10 +22,10 @@ let suites_1 = {
     param => ({
       TAG: "Eq",
       _0: true,
-      _1: typeof ({
+      _1: is_block({
```

Comment:
de-opted, but better result...? I think we can fold this into a single bool value here.